locales/zh/LC_MESSAGES/qmm.po

```diff
@@ -0,0 +1,372 @@
+
+#: qmm/manager.py:547
+msgid ""
+"Unable to uninstall mod: mismatched items exists on drive.\n"
+"This is most likely due to another installed mod conflicting with this mod.\n"
+msgstr ""
+"無法卸載模組：驅動器上存在不匹配的項目。\n"
+"這很可能是由於另一個已安裝的模組與此模組衝突。\n"
+
+#: qmm/manager.py:564
+#, python-brace-format
+msgid ""
+"The uninstallation process failed at some point. Please report this happened "
+"to the developper alongside with the error file {logfile}."
+msgstr ""
+"卸載過程在某些時候失敗。請將此情況報告給開發人員，並附上錯誤文件 {logfile}。"
+
+#: qmm/manager.py:674
+msgid "A file with the same name already exists in the repository."
+msgstr "存儲庫中已存在同名文件。"
+
+#: qmm/manager.py:695
+#, python-brace-format
+msgid ""
+"The file you selected is already present in the repository. It may exists "
+"under a different name.\n"
+"Hashsum matched: {hashsum}"
+msgstr ""
+"您選擇的文件已存在於存儲庫中。它可能以不同的名稱存在。\n"
+"哈希值匹配：{hashsum}"
+
+#: qmm/manager.py:708
+msgid "Install"
+msgstr "安裝"
+
+#: qmm/manager.py:710
+msgid "Uninstall"
+msgstr "卸載"
+
+#: qmm/manager.py:712
+msgid "Delete"
+msgstr "刪除"
+
+#: qmm/ui_about.py:66
+msgid "About"
+msgstr "關於"
+
+#: qmm/ui_about.py:67
+msgid "qModManager © 2020 Bicobus"
+msgstr "qModManager © 2020 Bicobus"
+
+#: qmm/ui_about.py:68
+msgid "Authors"
+msgstr "作者"
+
+#: qmm/ui_about.py:69
+msgid ""
+"qModManager is licensed under the European Union Public Licence V. 1.2\n"
+"\n"
+"EUPL © the European Union 2007, 2016\n"
+"\n"
+"This European Union Public Licence (the ‘EUPL’) applies to the Work (as\n"
+"defined below) which is provided under the terms of this Licence. Any use "
+"of\n"
+"the Work, other than as authorised under this Licence is prohibited (to the\n"
+"extent such use is covered by a right of the copyright holder of the Work).\n"
+"\n"
+"The Work is provided under the terms of this Licence when the Licensor (as\n"
+"defined below) has placed the following notice immediately following the\n"
+"copyright notice for the Work: “Licensed under the EUPL”, or has expressed "
+"by\n"
+"any other means his willingness to license under the EUPL.\n"
+msgstr ""
+"qModManager 根據歐盟公共許可證 V. 1.2 授權\n"
+"\n"
+"EUPL © 歐盟 2007, 2016\n"
+"\n"
+"本歐盟公共許可證（‘EUPL’）適用於根據本許可證條款提供的作品（如下所定義）。任何未經本許可證授權的使用均被禁止（在此類使用受作品版權持有人權利保護的範圍內）。\n"
+"\n"
+"當許可方（如下所定義）在作品的版權聲明後立即放置以下聲明：“根據 EUPL 授權”或以任何其他方式表達其願意根據 EUPL 授權時，作品即根據本許可證條款提供。\n"
+
+#: qmm/ui_about.py:343
+msgid "License"
+msgstr "許可證"
+
+#: qmm/ui_about.py:344
+msgid ""
+"qModManager make use of external libraries, framework and resources:\n"
+"\n"
+"PyQt5 - Copyright © 2019 Riverbank Computing Limited "
+"<info@riverbankcomputing.com> - GPLv3\n"
+"pyxdg - Thomas Kluyver, Heinrich Wendel, Sergey Kuleshov - GPLv2\n"
+"Remix Icon - https://remixicon.com - Apache License"
+msgstr ""
+"qModManager 使用外部庫、框架和資源：\n"
+"\n"
+"PyQt5 - 版權所有 © 2019 Riverbank Computing Limited "
+"<info@riverbankcomputing.com> - GPLv3\n"
+"pyxdg - Thomas Kluyver, Heinrich Wendel, Sergey Kuleshov - GPLv2\n"
+"Remix Icon - https://remixicon.com - Apache 許可證"
```

Comment:
Again, legalese shouldn't be translated.

The "Apache License" is the name of the license and ought to remain as is.